saleor/graphql/product/mutations/product/product_update.py

```diff
@@ -53,7 +52,8 @@ def clean_attributes(
     @classmethod
     def post_save_action(cls, info: ResolveInfo, instance, cleaned_input):
         product = models.Product.objects.prefetched_for_webhook().get(pk=instance.pk)
-        update_product_search_vector(instance)
+        product.search_index_dirty = True
+        product.save(update_fields=["search_index_dirty"])
```

Comment:
This might be not needed if we changed the `save` method of `ProductCreate` mutation (see comment above).